src/features/ar.ts

```diff
@@ -290,6 +317,16 @@ configuration or device capabilities');
           event.preventDefault();
           this.activateAR();
         }
+
+        get[$isAllowedQuickLookBrowser](): boolean {
```

Comment:
Nit, but I might call this $enableQuickLook, or $canLaunchQuickLook.

Reply:
Good suggestions, will fix 👍